File: 386-lexicographical-numbers/lexicographical-numbers.cpp
class Solution {
public:
    vector<int> lexicalOrder(int n) {
        vector<int> arr;
        vector<string> s(n);
        for(int i=1;i<=n;i++) {
            s[i-1] = to_string(i);
        }
        sort(s.begin(),s.end());
        for(int i=0;i<n;i++) {
            arr.push_back(stoi(s[i]));
        }
        return arr;
    }
};

File: 386-lexicographical-numbers/lexicographical-numbers.java
class Solution {
    public List<Integer> lexicalOrder(int n) {
        List<Integer> list = new ArrayList<>();
        String[] s = new String[n];
        for(int i = 1; i <= n ; i++){
            s[i-1] = Integer.toString(i);
        }
        Arrays.sort(s);
        for(int i = 0 ; i < n ; i++){
            list.add(Integer.parseInt(s[i]));
        }
        return list;
    }
}